src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -56,7 +56,7 @@ CREATE_LOGGERPTR_GLOBAL(logger_, "ProtocolHandler")
 std::string ConvertPacketDataToString(const uint8_t* data,
                                       const size_t data_size);
 
-const size_t kStackSize = 32768;
+const size_t kStackSize = 65536;
```

Comment:
> How easy is it to trigger this stack overflow?

With my testing environment (Ubuntu 16.04 x86_64 on VirtualBox, 3GB RAM), a crash always happens after proxy triggers StartService over Wi-Fi transport. Core crashes inside the constructor of CSmartSchema class, which is triggered during creation of a SmartObject in formatters::CFormatterJsonSDLRPCv2::toString(). (This is triggered by ApplicationManagerImpl::ConvertSOtoMessage() which is called by ApplicationManagerImpl::SendMessageToHMI()).

Actually I was also surprised when I found out the cause.